packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Settings/General/Language/list.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { Icon } from '@blockchain-com/constellation'
+import { IconCheckCircle } from '@blockchain-com/icons'
+
+import { selectors } from 'data'
+import { languagesSortedByName } from 'services/locales'
+
+import { CurrecyItemWrapper, CurrenciesList, CurrencySelectButton } from '../Currency/list'
+
+const List = (props) => {
+  const selectCurrency = (value) => {
+    if (value === props.language) return
+    props.input.onChange(value)
```

Comment:
I don't see `input` in props list. How does this work?

Reply:
It appears after connection to redux form by unique key